app/assets/javascripts/avalon/media-player.es6

```diff
@@ -105,11 +101,15 @@ export default class MediaPlayer {
     
     
     playFromHash() {
+        if (this.timer) {
+            clearTimeout(this.timer)
+            this.timer = null
+        }
         var mediaFragment = this.getMediaFragment(window.location.hash)
         var mediaPlayer = document.getElementById('iiif-av-player')
         mediaPlayer.setCurrentTime(mediaFragment.start)
         mediaPlayer.play()
-        setTimeout(() => {
+        this.timer = setTimeout(() => {
```

Comment:
@bkeese That was actually my fault. I discussed it this way with Jamie because I had in mind what we do for playlist and the scenario that has been discussed in IIIF-AV meetings (play just the _Famous Aria_). But in Avalon, you are right, we don't do this.

Sorry @little9